src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,56 @@
+.stop-watch {
+  @include circle(600px);
+  position: relative;
+  display: flex;
+  justify-content: center;
+
+  border: 5px solid $color-main-accent;
+
+  &__center {
+    @include circle(30px);
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    background-color: $color-main-accent;
+    z-index: 100;
+  }
+
+  &__minute-hand {
+    width: $hand-width;
+    height: 30%;
+
+    @include hand-position;
+
+    border-radius: $hand-radius;
+    background-color: #f75f28;
+    z-index: 50;
+
+    @include animation(3600s);
+  }
+
+  &__second-hand {
+    width: $hand-width;
+    height: 45%;
+
+    @include hand-position;
+
+    border-radius: $hand-radius;
+    background-color: #6495ed;
+
+    @include animation(60s);
+  }
+
+  @keyframes go-around {
+    0% {
+      transform: rotate(0);
+      transform-origin: bottom center;
```

Comment:
This property doesn't change during animation, so it is redundant or should be in another place

Reply:
When I delete 0% with its properties, it breaks. Or I didn't catch what you mean

Reply:
I mean transform-origin. It just repeated for 0% and 100%

Reply:
Thank you Ivan